README.md

```diff
@@ -40,5 +35,5 @@ With the [air-Q](https://www.air-q.com) you monitor the air quality in real time
 #### [Home Assistant](https://www.home-assistant.io/)
 * 🔗 https://forum.air-q.com/topic/137/iobroker-script/7
 
-#### [node red](https://nodered.org/)
+#### [homey](https://homey.app/)
```

Comment:
I think `Homey` is written with capital `H` in their self description.

Reply:
@dl2080 passt das so? :)